Documentation/ContentObjects/Fluidtemplate/DataProcessing/SiteLanguageProcessor.rst

```diff
@@ -1,35 +1,26 @@
-.. include:: /Includes.rst.txt
-.. _SiteLanguageProcessor:
+..  include:: /Includes.rst.txt
+..  _SiteLanguageProcessor:
 
-======================
+=====================
 SiteLanguageProcessor
-======================
+=====================
 
-.. versionadded:: 12.0
-   The Data Processor :php:`SiteLanguageProcessor` has been introduced, which
```

Comment:
I would keep some kind of introductionary text here

Reply:
The two paragraphs from before the patch are mostly the same. As they a redundant I removed one.